src/CommunityVoices/App/Website/View/Image.php

```diff
@@ -227,38 +227,6 @@ public function getAllImage($request)
 
     public function getImageUpload($request)
     {
-        $imageXMLElement = new SimpleXMLElement(
```

Comment:
Please entirely remove this route if you seek to take that route.

Reply:
@ethanmichel0 What ended up happening with this?  I see that it has been about two weeks - sorry for not checking in earlier.

Reply:
Actually, you were probably waiting for confirmation in one of the other comments. My bad.